File: C#/abc028A.cs
using System;

class main{
  public static int Main(){
    string a = Console.ReadLine();
    int b = int.Parse(a)
    if(b <= 59){
      Console.WriteLine("Bad");
    }
    if(60 <= b && b <= 89){
      Console.WriteLine("Good");
    }
    if(90 <= b && b <= 99){
      Console.WriteLine("Great");
    }
    if(100 == b){
      Console.WriteLine("Perfect");
    }
    return 0;
  }
}

File: C++/abc028A.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
  int a;
  cin >> a;
  if( a <= 59){
    cout << "Bad" << std::endl;
  }
  if(60 <= a && a <= 89){
    cout << "Good" << std::endl;
  }
  if(90 <= a && a <= 99){
    cout << "Great" << std::endl;
  }
  if(100 == a){
    cout << "Perfect" << std::endl;
  }
}
